src/Engine/Entity/World.cpp

```diff
@@ -85,10 +85,19 @@ Video::ParticleRenderer::Particle* World::GetParticles() const {
     return particles;
 }
 
+ENGINE_API unsigned int World::GetNrOfParticleSystems() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetNrOfParticleSystems' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)